tools/shoestring/shoestring/templates/docker-compose-dual.yaml

```diff
@@ -33,7 +33,11 @@ services:
       options:
         max-size: 25M
         max-file: '10'
+  {%- if recovery %}
```

Comment:
indenting of if blocks is inconsistent, in some places aligned with inner text in other places inner text is indented